src/DetailView/DetailView.js

```diff
@@ -1,53 +1,73 @@
+import { keyboard } from "@testing-library/user-event/dist/keyboard"
 import React, { Component } from "react"
 import "./DetailView.css"
+import backButton from "../assets/back-button.png"
 
 class DetailView extends Component {
   constructor(props) {
     super(props)
     this.movieId = this.props.movieId
     this.closeMovie = this.props.closeMovie
-    this.state = {
-    }
+    this.state = {}
   }
 
   componentDidMount() {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.movieId}`)
       .then(response => response.json())
       .then(movieObject => {
         const data = movieObject.movie
+        const releaseDateObject = new Date(Date.parse(data.release_date))
+        const dateData = `${releaseDateObject.toLocaleString("default", { month: "long" })} ${releaseDateObject.getDate()}, ${releaseDateObject.getFullYear()}`
+        const dateYear = releaseDateObject.getFullYear()
 
         this.setState({
           averageRating: data.average_rating,
           backdropPath: data.backdrop_path,
-          budget: data.budget,
-          genres: data.genres,
+          budget: data.budget.toLocaleString(),
+          genres: data.genres.join(' '),
```

Comment:
Joining the genres together with a space looks a little crowded in the browser. Maybe we could treat each genre as a separate `<p>` tag (since in a professional UI, they would likely be linking to genre pages), or at least separate them by an extra space or two?

Reply:
Sure! I can wrap them in a container and make them a separate class so we can reference them down the road if we want to be able to enable some functionality based around genre tags.

Reply:
I ended up joining them with " | ". I do like the idea of using <p> tags instead, but I'm a bit worn out for the day and it's closing in on 4pm here so I'm just going to call it good enough for this weekend. Definitely open to refactoring this down the road though, just not tonight.